src/pathfinder.rs

```diff
@@ -1,3 +1,14 @@
+//! Manually accessing the [`PathFinder`] API
+//!
+//! This contains functionality from the [`PathFinder`] object in Screeps, which
+//! is itself a binding to a C++ implementation of the astar algorithm backed by
```

Comment:
Isn't it a jump-point algorithm?

Reply:
Ah - I hadn't heard of that algorithm - it totally is!